source/server/hot_restart_impl.cc

```diff
@@ -473,7 +474,7 @@ void HotRestartImpl::terminateParent() {
 
 void HotRestartImpl::shutdown() { socket_event_.reset(); }
 
-std::string HotRestartImpl::version() { return shmem_.version(); }
+std::string HotRestartImpl::version() { return shmem_.version() + stats_set_.version(); }
```

Comment:
Should we put a separator between the two?  space or period?

Reply:
Yeah, sorry, will do.